src/modules/comment.js

```diff
@@ -62,11 +62,11 @@ export default class Comment {
     const commentDisplay = document.querySelector('.comment-display');
     commentDisplay.innerHTML = '';
     const commentCounter = document.querySelector('.comment-counter');
+    commentCounter.textContent = `Comments (${this.commentCounter(
+      gameComments,
+    )})`;
```

Comment:
Well done moving the commentCounter element outside the if statement, so that even when there is no comment it is still updated, as such.